src/Nox.Types/Types/PhoneNumber/PhoneNumber.cs

```diff
@@ -9,6 +9,7 @@ namespace Nox.Types;
 public sealed class PhoneNumber : ValueObject<string, PhoneNumber>
 {
     private static readonly Regex _phoneNumberRegex = new(@"^(?:\+\d{1,3}[\s\-]?)?(?:\(\d{1,4}\)|\d{1,4})\s*\-?\s*(?:\d{1,4}\s*\-?\s*){1,4}$", RegexOptions.Compiled, TimeSpan.FromSeconds(1));
+    private static readonly int _maxPhoneNumberLength = 30;
```

Comment:
can we make this public and reuse the value on the DatabaseConfigurator?

Reply:
resolving as done